.github/workflows/ci-cd.yml

```diff
@@ -249,7 +249,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           # Used for computing the Android versionCode https://github.com/CruGlobal/missionhub-react-native/blob/4c37730c21baeea3a304702b1bbbbce3d11f320d/android/app/build.gradle#L145
-          fetch-depth: 0
+          fetch-depth: 2
```

Comment:
I think this was supposed to change on the `test` job, not the `android-deploy`

Reply:
@wrandall22 my bad already added the fetch-depth to the test